sql/core/src/test/resources/sql-tests/inputs/group-by.sql

```diff
@@ -30,8 +30,15 @@ SELECT a + 2, COUNT(b) FROM testData GROUP BY a + 1;
 SELECT a + 1 + 1, COUNT(b) FROM testData GROUP BY a + 1;
 
 -- Aggregate with nulls.
+--
+-- In SPARK-21871, we added code to check the bytecode size of gen'd methods. If the size
```

Comment:
If the issue in #19082 fixed, we might remove workaround. Or, we might use a new flag discussed in #19062 here.